### ISSUE TITLE
Favicon injection and favicons-webpack-plugin

### ISSUE THREAD

**Issue by mikevaleriano** [arthurbergmz/webpack-pwa-manifest#85]
I successfully generated a PWA manifest and all there's left to do is link the icons in my html.

After finding #22 and #41 I got a bit confused: it seems I would have to use [favicons-webpack-plugin](https://github.com/jantimon/favicons-webpack-plugin) to generate and inject the links to the generated icons in my html, but I don't get how the icons will then be listed in the manifest generated by this plugin.

I thought 4.0.0 was coming soon, and proper favicon support would be there, but then I found #70, and it may not be the case.

**TL;DR:** Should I not generate icons using this plugin? If not, how do I automatically reference the icons in the generated manifest when they are created by another plugin?

**Comment by markusgattol** [comment]
any updates on this i.e. how do people inject their icons? manually still?

**Comment by jalik** [comment]
I also think that using one plugin to generate icons + manifest and injecting them into the HTML is the way to go ! since this plugin is already generating icons and manifest, maybe it's not a big deal to also inject favicons in the HTML ?
